src/WebJobs.Script/Binding/WebJobsCoreScriptBindingProvider.cs

```diff
@@ -102,45 +69,46 @@ public override Collection<Attribute> GetAttributes()
             }
         }
 
-        private class BlobScriptBinding : ScriptBinding
-        {
-            public BlobScriptBinding(ScriptBindingContext context) : base(context)
-            {
-            }
-
-            public override Type DefaultType
-            {
-                get
-                {
-                    return typeof(Stream);
-                }
-            }
-
-            public override Collection<Attribute> GetAttributes()
-            {
-                Collection<Attribute> attributes = new Collection<Attribute>();
-
-                string path = Context.GetMetadataValue<string>("path");
-                Attribute attribute = null;
-                if (Context.IsTrigger)
-                {
-                    attribute = new BlobTriggerAttribute(path);
-                }
-                else
-                {
-                    attribute = new BlobAttribute(path, Context.Access);
-                }
-                attributes.Add(attribute);
-
-                var connectionProvider = (IConnectionProvider)attribute;
-                string connection = Context.GetMetadataValue<string>("connection");
-                if (!string.IsNullOrEmpty(connection))
-                {
-                    connectionProvider.Connection = connection;
-                }
-
-                return attributes;
-            }
-        }
+        // TODO: DI (FACAVAL) Load storage extensions dynamically
```

Comment:
What work remains for this - blob binding doesn't work as is? I assume the answer is that the GeneralScriptBindingProvider is supposed to handle all of these?